Cineaste/ViewController/MyMovies/MoviesViewController+SearchController.swift

```diff
@@ -0,0 +1,27 @@
+//
+//  MoviesViewController+SearchController.swift
+//  Cineaste
+//
+//  Created by Felizia Bernutz on 14.07.18.
+//  Copyright © 2018 notimeforthat.org. All rights reserved.
+//
+
+import UIKit
+
+extension MoviesViewController: UISearchResultsUpdating {
+    func updateSearchResults(for searchController: UISearchController) {
+        guard let searchText = searchController.searchBar.text else { return }
+
+        if !searchText.isEmpty {
+            let predicate = NSPredicate(format: "title contains[c] '\(searchText)'")
```

Comment:
Isn't this the same problem as before? The predicate does not consider on which list we currently are